src/createReactQueryClient.js

```diff
@@ -0,0 +1,16 @@
+import { QueryClient } from 'react-query';
+
+const cacheTimeMinutes = 5;
+
+const createReactQueryClient = () => new QueryClient({
```

Comment:
In order to access the client in the individual applications, the [`useQueryClient`](https://react-query.tanstack.com/reference/useQueryClient#_top) hook should be used so there is no need to export the client from the stripes-core.